programs/bpf_loader/src/lib.rs

```diff
@@ -4033,39 +4033,39 @@ mod tests {
         // when `enable_heap_size_round_up` not enabled:
         {
             // assert less than 32K heap should cost zero unit
-            assert_eq!(0, calculate_heap_cost(31_u64 * 1024, heap_cost, false));
+            assert_eq!(0, calculate_heap_cost(31_u32 * 1024, heap_cost, false));
```

Comment:
let's just drop all these `_u32` suffixes; the function takes in a `u32` so these literals are automatically `u32`